src/services/completions.ts

```diff
@@ -1594,6 +1594,7 @@ namespace ts.Completions {
 
         function tryGetImportCompletionSymbols(): GlobalsSearch {
             if (!importCompletionNode) return GlobalsSearch.Continue;
+            isNewIdentifierLocation = true;

```

Comment:
I discovered this was missing only because I added fourslash server tests... unrelated to main change